src/main/java/com/github/swapbook/email/EmailServiceImpl.java

```diff
@@ -0,0 +1,31 @@
+package com.github.swapbook.email;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.mail.SimpleMailMessage;
+import org.springframework.mail.javamail.JavaMailSender;
+import org.springframework.stereotype.Component;
+
+@Component
+public class EmailServiceImpl implements EmailService {
+
+    private static final Logger log = LoggerFactory.getLogger(EmailServiceImpl.class);
+
+    @Autowired
+    private JavaMailSender emailSender;
+
+    public void sendSimpleMessage(String to, String subject, String text) {
```

Comment:
zmień nazwe "to" bo jest totalnie nie jasna moze lepiej było by userMail albo mailiTo